src/cursor/cur_std.c

```diff
@@ -1067,6 +1090,14 @@ __wt_cursor_reconfigure(WT_CURSOR *cursor, const char *config)
             if (WT_CURSOR_RECNO(cursor))
                 WT_ERR_MSG(
                   session, EINVAL, "cannot use prefix key search near for column store formats");
+            /*
+             * Prefix search near configuration can only be used for formats string or raw byte
+             * array.
```

Comment:
This sentence sounds awkward to me, maybe `for string or raw byte array formats`? Feel free to ignore if I'm  wrong!